### PR TITLE
Add controls for handling video exit

### LINKED ISSUE #75: exit video cleanly when playback finished
maybe 2 buttons:
* replay
* close

### PR DESCRIPTION
**Pull request by meandavejustice**
- adds "replay" and "close" buttons when video finishs
- sends "replay" event to metrics
- adds "played_count" to metrics
- fixes #75
- bumped package version to `0.1.1`

Screenshot: (Close button shows `:hovered` behavior)
min-vid-exit-ui-updated

Still need to submit a bugzilla bug like https://bugzilla.mozilla.org/show_bug.cgi?id=1270586 for redshift to be able to handle the new `played_count` metric

### CODE REVIEW

**Comment by meandavejustice** [comment]
Adding updated icons from #75 today.

**Comment by 6a68** [comment]
@meandavejustice taking a look now. I don't think the redshift schemas are actually baked yet, so I think we might skate by without needing to file an update bug. @wresuolc would know for sure

**Comment by 6a68** [comment]
@meandavejustice I'm not seeing the new UI when videos end. Is there anything I need to do?

**Comment by 6a68** [comment]
Whoops! I had some dirty changes in the index. Working for me now, looking at the code

File: app.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const defaultData = {
   duration: 0,
   progress: 0.001, // force progress element to start out empty
   playing: false,
+  'played_count': 0,
```
Comment (6a68) [inline app.js]: I'm not a fan of mixing snake_case and camelCase, could we just make this playedCount?

File: docs/metrics.md [hunk 2]
```diff
@@ -78,6 +81,8 @@ to the number of seconds since Firefox app startup, and wraps the payload under
     "method": "pause",               // Event type
 
     "domain": "youtube.com",         // Domain from a whitelist of video hosting sites
+    "played_count": "2",             // count of full video playback
```
Comment (6a68) [inline docs/metrics.md]: The 2 shouldn't be quoted

File: docs/metrics.md [hunk 3]
```diff
@@ -109,6 +114,7 @@ local schema = {
     {"object",                     "VARCHAR",   255,     nil,         "payload[object]"},
     {"method",                     "VARCHAR",   255,     nil,         "payload[method]"},
     {"domain",                     "VARCHAR",   255,     nil,         "payload[domain]"},
+    {"played_count",               "VARCHAR",   255,     nil,         "payload[played_count]"},
```
Comment (6a68) [inline docs/metrics.md]: It might be good to use a numeric type for this data

File: docs/metrics.md [hunk 4]
```diff
@@ -109,6 +114,7 @@ local schema = {
...
 
     {"video_x",                    "BOOLEAN",   nil,     nil,         "payload[video_x]"},
     {"video_y",                    "BOOLEAN",   nil,     nil,         "payload[video_y]"},
```
Comment (6a68) [inline docs/metrics.md]: Now that I look at this file, it seems like a mistake that these video_* keys are set to be a Boolean type. Would you mind updating these to be numbers?

**Comment by 6a68** [comment]
@meandavejustice Looks good! Couple of really minor things, feel free to merge once addressed

File: app.js (hunk 1 above)
Comment (meandavejustice) [inline app.js]: np

File: docs/metrics.md (hunk 2 above)
Comment (meandavejustice) [inline docs/metrics.md]: thanks, I'll fix that up

File: docs/metrics.md (hunk 4 above)
Comment (meandavejustice) [inline docs/metrics.md]: sure

**Comment by meandavejustice** [comment]
@6a68 Do I actually need to submit a bugzilla bug yet for this change? I don't see an initial bug for min-vid metrics in bugzilla

**Comment by meandavejustice** [comment]
@6a68  thanks for the review

**Pull request merged**